packages/dtslint/src/lint.ts

```diff
@@ -112,11 +112,11 @@ function testDependencies(
     const typesFolder = dirname(path);
 
     return `
-A module look-up failed, this often occurs when you need to run \`npm install\` on a dependent module before you can lint.
+A module look-up failed, this often occurs when you need to run \`pnpm install\` on a dependent module before you can lint.
 
 Before you debug, first try running:
 
-   npm install --prefix ${typesFolder}
+   pnpm install --filter '{./types/${typesFolder}}...'
```

Comment:
I'm actually somewhat suspicious of this rule anymore; what we should be telling people is to run `pnpm install -w --filter '...{./types/what-they-are-working-on}...'`, right? 

Or maybe it's unlikely people hit this at all anymore?